src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useContext } from 'react';
+import { TodoItem } from '../TodoItem';
+import { Todo } from '../../types/Todo';
+import { removeTodo } from '../../api/todos';
+import { DeleteModalContext } from '../../context/DeleteModalContext';
+
+type Props = {
+  todos: Todo[];
+  setTodos: React.Dispatch<React.SetStateAction<Todo[]>>;
+  deleteTodo: (id: number) => void;
+};
+
+export const TodoList: React.FC<Props> = ({
+  todos,
+  setTodos,
+  deleteTodo,
+}) => {
+  const { setDeleteModal } = useContext(DeleteModalContext);
+
+  const updateTodo = (
+    event: React.ChangeEvent<HTMLInputElement>,
+    id: number,
+  ) => {
+    const { checked } = event.target;
+
+    setTodos((prevState: Todo[]) => {
+      return prevState.map(todo => {
+        if (todo.id === id) {
+          return {
+            ...todo,
+            completed: checked,
+          };
+        }
+
+        return todo;
+      });
+    });
+  };
+
+  const clearTodo = (todo: Todo) => {
+    setDeleteModal([todo.id]);
```

Comment:
why I don't see modal after clear or delete todo?

Reply:
![2023-07-27_15h16_41](https://github.com/mate-academy/react_todo-app-add-and-delete/assets/61904995/ac484546-7785-451b-a88d-1d41396e3a2b)
I dont find modal at all, in any cases. Why you need `deleteModal` if you don't use it?